bin/run_tests

```diff
@@ -341,11 +338,6 @@ test_rust_sdk() {
 }
 
 test_integration() {
-    run_docker_test test_xo_smoke_all
-    copy_coverage .coverage.test_xo_smoke_all
-    run_docker_test test_web_socket_subscription
-    run_docker_test test_poet_smoke
```

Comment:
Taking out poet smoke should be in a separate commit (perhaps even a separate PR), since it doesn't pertain to restructuring the repo.

That said, if you're culling outdated tests, I suggest also getting rid of the shutdown smoke test (    https://github.com/hyperledger/sawtooth-core/blob/647f9566c8cf104cf0dc9e16404fa432d4dfd3a8/bin/run_tests#L271). It hasn't run in quite some time (the test container starts, but the actual Python tests are skipped), and it would take a fair amount of effort to get it to work.

Reply:
I moved the removal of these into a separate commit.